pyproject.toml

```diff
@@ -10,25 +10,17 @@ requires-python = ">=3.10"
 dynamic = ["version", "description"]
 
 dependencies = [
-  "submitit>=1.5.2",
   "psutil",
-  "torch>=2.5.1",
-  "transformers>=4.45.2",
-  "vllm>=v0.6.6.post1",
-  "ray[serve]>=2.40.0",
```

Comment:
removing all these packages will save a lot of time during installation (and potentially more resolving conflicting versions) !